builtin/logical/transit/path_sign_verify.go

```diff
@@ -5,12 +5,57 @@ import (
 	"encoding/base64"
 	"fmt"
 
+	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/vault/helper/errutil"
 	"github.com/hashicorp/vault/helper/keysutil"
 	"github.com/hashicorp/vault/logical"
 	"github.com/hashicorp/vault/logical/framework"
+	"github.com/mitchellh/mapstructure"
 )
 
+// BatchRequestSignItem represents a request item for batch processing.
+// A map type allows us to distinguish between empty and missing values.
+type batchRequestSignItem map[string]string
+
+// BatchResponseSignItem represents a response item for batch processing
+type batchResponseSignItem struct {
+	// signature for the input present in the corresponding batch
+	// request item
+	Signature string `json:"signature,omitempty" structs:"signature" mapstructure:"signature"`
```

Comment:
Here and elsewhere, if you are not using the `structs` package in your tests for this functionality, please don't decorate it. We are generally trying to get rid of usage of that package.